packages/inter-protocol/src/proposals/README.md

```diff
@@ -1,7 +1,24 @@
 # Proposals
 
-These are code snippets that go into propoals to the BLDer DAO to start the Inter Protocol.
+These scripts are referenced by proposals to the BLDer DAO to run on the chain.
```

Comment:
This documentation is good. It strikes me as not specific to inter-protocol.

Consider putting it in deploy-script-support and referencing that here.

Reply:
Good idea.